packages/react/src/hooks/useOkto.ts

```diff
@@ -10,3 +10,8 @@ export function useOkto(): OktoClient {
 
   return client;
 }
+
+export function useAuthWebView() {
+  const client = useOkto();
+  return client['authWebView']; // Access the private `authWebView` instance
+}
```

Comment:
@SrijanCoinDCX why two functions with similar names ?

`useWebViewAuth()`  and `useAuthWebView()`

Reply:
@SrijanCoinDCX remove this